src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -37,6 +38,13 @@ public SortCommand parse(String args) throws ParseException {
         }
 
         List<SortArgument> sortArgs = convertPrefixArgToSortArg(argList);
+
+        for (SortArgument arg : sortArgs) {
+            if (arg == null) {
+                throw new ParseException(Tag.MESSAGE_CONSTRAINTS);
+            }
```

Comment:
Should empty strings be guarded against as well?

Reply:
Empty strings are guarded against in the `parseTag` method which is called in `convertArguments`. Hence, the `null` object here actually guards against **all** invalid tag names.

Reply:
Oh I see. Okay.